pkg/loader/loader.go

```diff
@@ -404,7 +404,7 @@ func LoadRootsWithConfig(cfg *packages.Config, roots ...string) ([]*Package, err
 	loadPackages := func(roots ...string) ([]*Package, error) {
 		rawPkgs, err := packages.Load(l.cfg, roots...)
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf("error in %s: %w", l.cfg.Dir, err)
```

Comment:
`l.cfg.Dir` is allowed to be empty string (in which case the loader loads from the current working directory). I'm not sure that's possible in c-t's use of this package as it is currently written, but either way, I think we should account for this possibility if we're going to print its value.

Also, maybe even more context would have been helpful here?
```suggestion
			loadRoot := l.cfg.Dir
			if l.cfg.Dir == "" {
				loadRoot, _ = os.Getwd()
			}
			return nil, fmt.Errorf("load packages in root %q: %w", loadRoot, err)
```